src/config/mod.rs

```diff
@@ -91,7 +95,7 @@ pub fn safe_apps_cache_duration() -> usize {
 }
 
 pub fn token_price_cache_duration() -> usize {
-    env_with_default("TOKEN_PRICE_CACHE_DURATION", 1) // set to negligible value
+    env_with_default("TOKEN_PRICE_CACHE_DURATION", 10 * 1000)
```

Comment:
Should we do the test stuff @jpalvarezl mentioned for this one too?

Reply:
I added a default value test here f6b379a